README.md

```diff
@@ -71,52 +65,41 @@ then configure it.
 npm run web-dev
 ```
 
-Wowser will be served on `http://localhost:8080`.
+Wowser will be served on `http://0.0.0.0:8080/webpack-dev-server/`.
 
-### Pipeline server
+### Socket proxies
 
-To deliver game resources to its client, Wowser ships with a pipeline.
+To utilize raw TCP connections a WebSocket proxy is required for JavaScript
+clients.
 
-Build the pipeline:
+[Websockify] can - among other things - act as a proxy for raw TCP sockets.
 
-```shell
-npm run gulp
-```
+For now, you will want to proxy both port 3724 (auth) and 8085 (world). 
 
-Keep this process running to monitor source files and automatically rebuild.
+#### - Proxy port in localhost
 
-After building, serve the pipeline as follows in a separate process:
+If you want to connect this web client to a server in the same machine you can change the authserver port from 8085 to 8086 in the  **auth realmlist table** using:
+```SQL
+UPDATE `realmlist` SET `port`=8086 WHERE `id`=1;
+```
 
+Run proxy for localhost:
 ```shell
-npm run serve
+npm run proxy 3725 127.0.0.1:3724
+npm run proxy 8086 127.0.0.1:8085
 ```
 
-On first run you will be prompted to specify the following:
-
-- Path to client data folder
-- Server port (default is `3000`)
-- Number of cluster workers (default depends on amount of CPUs)
+While login using the web client, use the port 3725.
 
-Clear these settings by running `npm run reset`
-
-**Disclaimer:** Wowser serves up resources to the browser over HTTP. Depending
-on your network configuration these may be available to others. Respect laws and
-do not distribute game data you do not own.
-
-### Socket proxies
-
-To utilize raw TCP connections a WebSocket proxy is required for JavaScript
-clients.
-
-[Websockify] can - among other things - act as a proxy for raw TCP sockets.
-
-For now, you will want to proxy both port 3724 (auth) and 8129 (world). Use a
-different set of ports if the game server is on the same machine as your client.
+#### - Proxy port for a public server
 
+Run proxy for a public server:
 ```shell
-npm run proxy 3724 host:3724
-npm run proxy 8129 host:8129
+npm run proxy 3724 server.realmlist:3724
+npm run proxy 8085 server.realmlist:8085
 ```
+replacing *server.realmlist* with the realmlist of the server.
```

Comment:
We could add "the trick" to test it locally here

Reply:
I already added it above